ConfigurationSystem/Client/PathFinder.py

```diff
@@ -30,29 +30,45 @@ def getSystemSection( serviceName, serviceTuple = False, instance = False, setup
     instance = getSystemInstance( serviceTuple[0], setup = setup )
   return "/Systems/%s/%s" % ( serviceTuple[0], instance )
 
+def getComponentSection (componentName, componentTuple = False, setup = False, componentCategory ="Services"):
+  """Function returns the path to the component.
```

Comment:
It is google docstring-like, so it should be ok if the sphinx is able to parse google format properly, and I hope too